js/app.js

```diff
@@ -0,0 +1,42 @@
+//put app in expert mode
+"use strict"
+
+//begin to define main app.js
+//IIFE - keeps the app from having many global variables
+(function() {
+  angular
+//"wdiRadio" is my first argument and the name of the app
+//the name is followed by the second argument, which is the function that encloses all functionality of the controller
+  .module("wdiRadio", [
+//injected ui.router into main module
+    "ui.router",
+//inject "songs" into app.js
+    "songs",
+    "ng-token-auth"
```

Comment:
Were you planning to add user authentication to this assignment? I don't think we went over `ng-token-auth` in any of our angular lessons, it can definitely be tricky!